doc/specific_iocs/dae/Datastreaming.md

```diff
@@ -23,30 +23,56 @@ Part of our in-kind contribution to datastreaming is to test the system in produ
+## The Kafka Cluster
 
-### Grafana dashboard
-A Grafana dashboard for the production cluster can be found at `madara.isis.cclrc.ac.uk:3000`. This shows the topic data rate and other useful information. Admin credentials can also be found in the sharepoint. 
+There is a Kafka cluster at `livedata.isis.cclrc.ac.uk`. Port 9092 is used for the primary Kafka broker. A web interface
+is available on port 8080.
+
+The production server auto-creates topics when those topics are produced to; consuming however does not create them.
+
+Credentials for the cluster can be found in Keeper, under `ds streaming container user`. The machine is reachable by
+SSH with these credentials.
 
 ### Deployment
-Deployment involves the use of Ansible playbooks, the playbooks and instructions for using these can be found [here.](https://github.com/ISISComputingGroup/ansible-kafka-centos)
+
+Deployment is currently onto a machine running in the SCD cloud. Deployment instructions can be found
+[in the `ds-containers` repository](https://github.com/isiscomputinggroup/ds-containers).
 
 ## Neutron Data
-The ICP on any instrument that is running in full event mode and with a DAE3 is streaming neutron events into Kafka. 
+
+The ICP on any instrument that is running in full event mode and with a DAE3 may stream neutron events into Kafka.
+
+This is controlled using flags in the `isisicp.properties` file:
+
+```
+isisicp.kafkastream = true
+# if not specified, topicprefix will default to instrument name in code
+isisicp.kafkastream.topicprefix =
+isisicp.kafkastream.broker = livedata.isis.cclrc.ac.uk:9092
+isisicp.kafkastream.topic.suffix.runinfo = _runInfo
+isisicp.kafkastream.topic.suffix.sampleenv = _sampleEnv
+isisicp.kafkastream.topic.suffix.alarms = _alarms
+```
 
 ## SE Data
+
 See [Forwarding Sample Environment](datastreaming/Datastreaming---Sample-Environment)
 
 ## Filewriting
 
 See [File writing](datastreaming/Datastreaming---File-writing)
 
 ## System Tests
-Currently system tests are being run to confirm that the start/stop run and event data messages are being sent into Kafka and that a Nexus file is being written with these events. The Kafka cluster and filewriter are being run in docker containers for these tests and so must be run on a Windows 10 machine. To run these tests you will need to install [docker for windows and add yourself as a docker-user](https://docs.docker.com/docker-for-windows/install/#install-docker-desktop-on-windows).
+
+Currently system tests are being run to confirm that the start/stop run and event data messages are being sent into
```

Comment:
These aren’t run any more, but we should write some again that test data streaming